addons/planner_project/data/planner_data.xml

```diff
@@ -0,0 +1,1211 @@
+                            <span class="fa fa-fw"/>Objectives
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page3">
+                            <span class="fa fa-fw"/>Activities
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page4">
+                            <span class="fa fa-fw"/>Projects
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page5">
+                            <span class="fa fa-fw"/>Kanban Stages
+                        </a>
+                    </li>
+                    <h4><i class="fa fa-cogs"></i> Implement</h4>
+                    <li>
+                        <a href="#planner_page6">
+                            <span class="fa fa-fw"/>Tasks &amp; Issues
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page7">
+                            <span class="fa fa-fw"/>Timesheets
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page8">
+                            <span class="fa fa-fw"/>Customization
+                        </a>
+                    </li>
+                     <li>
+                        <a href="#planner_page9">
+                            <span class="fa fa-fw"/>Get more Apps
+                        </a>
+                    </li>
+                    <h4><i class="fa fa-rocket"></i> Deploy</h4>
+                    <li>
+                        <a href="#planner_page10">
+                            <span class="fa fa-fw"/>Responsibilities
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page11">
+                            <span class="fa fa-fw"/>Notifications
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page12">
+                            <span class="fa fa-fw"/>Invite Users
+                        </a>
+                    </li>
+                    <h4><i class="fa fa-book"></i> Improve</h4>
+                    <li>
+                        <a href="#planner_page13">
+                            <span class="fa fa-fw"/>Organize Meetings
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#planner_page14">
+                            <span class="fa fa-fw"/>Generate Reports
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+            <!--Planner Pages-->
+            <div class="pages col-sm-8 col-md-9 col-xs-12">
+
+                <div id="planner_page1" class="planner-page visible">
```

Comment:
rebase your branch, I change classes : now using bootstrap `show` class instead of visible ;)